server/src/test/java/io/crate/integrationtests/HasTablePrivilegeFunctionIntegrationTest.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.integrationtests;
+
+import static io.crate.metadata.pgcatalog.OidHash.Type.fromRelationType;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.elasticsearch.test.IntegTestCase;
+import org.junit.Test;
+
+import io.crate.metadata.RelationInfo;
+import io.crate.metadata.RelationName;
+import io.crate.metadata.pgcatalog.OidHash;
+
+public class HasTablePrivilegeFunctionIntegrationTest extends IntegTestCase {
```

Comment:
Do you think it's important to have an integ test for this? Is there anything more in the code path that is tested with this?
I'd say at least if we keep it, do it in one method to not repeat the setup/tear down.

Reply:
Since name -> oid and oid-> name conversions are slightly different for different types of relations and also I suspect there could be some refactoring around oids (i.e. impvoring old -> name which is very inefficient at the moment) I would like to keep it.
I am merging them as a single test as you suggest, Thank you.